tests/testapp/models.py

```diff
@@ -24,6 +24,7 @@
 
 
 class ModelWithVanillaMoneyField(models.Model):
+    # TODO: This should now be an error?
```

Comment:
@benjaoming I think this TODO should be addressed.

Reply:
Could you, please, elaborate a bit?

Reply:
@Stranger6667 I think my original reasoning here is that it should be an error to provide a default value without a currency. Regardless of whether or not that makes sense, I should the comment should either be acted on or removed.

Reply:
This field now raises an error due to #661 being merged. i.e. this has to be corrected, it no longer makes sense to have a default value that can't be converted to a `Money` instance (-> needs a currency default or value that isn't `None`)

Reply:
While refreshing my memory I think this is where I built #661 from, while attempting to move this PR forward. Though I chose to open a separate PR as it needed functional changes which weren't depending on changing the default currency to `None`.

I'm pretty what should happen here is to remove `default=0.0`

Reply:
@flaeppe Your reasoning here sounds exactly right to me 👍

I added you as collaborator on my fork, so you should be able to push to this branch. But also feel free to replace this PR with a fresh one :)

Reply:
Cool, thanks @antonagestam. I think I managed to merge `master` properly in to this branch.

Reply:
Should be possible to resolve this comment. The `TODO` is gone together with `default=0.0` and affected tests have been adjusted.